## request fields
string dest_ip_addr
---
## response fields
float32 packet_loss
float32 avg_time         # ms